src/ResourceManager/Network/Commands.Network/Models/PSPeering.cs

```diff
@@ -31,7 +31,7 @@ public class PSPeering : PSChildResource
         public int AzureASN { get; set; }
 
         [JsonProperty(Order = 1)]
-        public int PeerASN { get; set; }
+        public uint PeerASN { get; set; }
```

Comment:
Was it previously possible for the user to input a negative value and not have an error thrown at runtime? I just want to make sure that any scripts will not be broken by this change.